jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -121,6 +129,15 @@ def replace_docs(request, docs):
         """
         request.data.docs = docs
 
+    @staticmethod
+    def replace_parameters(request, parameters):
```

Comment:
type annotations please

Reply:
it seems that none of the static method of the DataRequestHandler are typed annotated, should I add type annotation for the method that are out of the scope of this PR as well ? ( for replace_docs and replace_parameters)

Reply:
If the type replace_docs and merge_routes should not be added in this PR I can revert (https://github.com/jina-ai/jina/pull/4224/commits/c9a3267fb87df50d605c1472f2c6585424ef5e76)